src/fides/data/test_env/fides.test_env.toml

```diff
@@ -22,6 +22,7 @@ oauth_root_client_id = "fidesadmin"
 oauth_root_client_secret = "fidesadminsecret"
 root_username = "root_user"
 root_password = "Testpassword1!"
+drp_jwt_secret = "secret"
```

Comment:
I think this one could still be optional since we're not redefining it and just importing the model from fideslib